lte/gateway/docker/docker-compose.yaml

```diff
@@ -19,6 +19,8 @@ x-standard-volumes: &volumes_anchor
   - /tmp:/tmp
   - /var/log:/var/log
   - /etc/openvswitch:/etc/openvswitch
+  - /home/vagrant/magma/lte/gateway/configs:/magma/lte/gateway/configs
```

Comment:
Shouldn't this be covered by `${CONFIGS_OVERRIDE_VOLUME}:/var/opt/magma/configs`?

Reply:
Would it be an option to copy the changed configs to `${CONFIGS_OVERRIDE_VOLUME}` (usually /var/opt/magma/configs)?

Reply:
As discussed with @crasu services probably shouldn't read from `/magma/lte/gateway/configs` anyway. If they insist to do so and it's hard to change, we might keep it for now but create a ticket to improve that.

Reply:
After some testing it is clear that the services actually don't read the configs from `/magma/lte/gateway/configs`, but rather from `/etc/magma` (which are mounted as part of the `$(CONFIGS_DEFAULT_VOLUME)`). I removed the additional mount, because it is unnecessary and unused.